test/src/org/labkey/test/components/targetedms/QCPlotsWebPart.java

```diff
@@ -284,13 +284,20 @@ public List<QCPlot> getPlots()
         return plots;
     }
 
-    public String getSVGPlotText(String id)
+    public String getSVGPlotText(int plotPanelIndex, String plotIdSuffix)
```

Comment:
This function doesn't use plotPanelIndex?

Reply:
Oops